server/src/test/java/org/elasticsearch/transport/RemoteClusterSettingsTests.java

```diff
@@ -71,9 +76,48 @@ public void testSeedsDefault() {
         assertThat(REMOTE_CLUSTER_SEEDS.getConcreteSettingForNamespace(alias).get(Settings.EMPTY), emptyCollectionOf(String.class));
     }
 
+    public void testCredentialsDefault() {
+        assumeTrue("Skipped test because CCx2 feature flag is not enabled", TcpTransport.isUntrustedRemoteClusterEnabled());
+        final String alias = randomAlphaOfLength(8);
+        assertThat(REMOTE_CLUSTER_AUTHORIZATION.getConcreteSettingForNamespace(alias).get(Settings.EMPTY), equalTo(""));
+    }
+
     public void testProxyDefault() {
         final String alias = randomAlphaOfLength(8);
         assertThat(REMOTE_CLUSTERS_PROXY.getConcreteSettingForNamespace(alias).get(Settings.EMPTY), equalTo(""));
     }
 
+    public void testRemoteClusterEmptyOrNullApiKey() {
+        assumeTrue("Skipped test because CCx2 feature flag is not enabled", TcpTransport.isUntrustedRemoteClusterEnabled());
+        // simple validation
+        Settings settings = Settings.builder()
+            .put("cluster.remote.cluster1.authorization", "apikey")
+            .put("cluster.remote.cluster2.authorization", "")
```

Comment:
We should probably validate that we *don't* accept an empty string. (null is acceptable as a way to unset)

Reply:
I think that is in core settings code. If not set, the setting value returns an empty string.

Reply:
in that case, we probably don't need to explicitly test behaviors inherited from core setting code. If we have custom validation or deviate from core setting capabilities then we want to explicitly test.  (a bit surprised we get back empty string instead of null if not set...but it doesnt matter much)

Reply:
I removed the empty setting value test.